apiary-metastore-listener/README.md

```diff
@@ -65,7 +75,11 @@ The following shows an example JSON message representing an "ALTER_TABLE" event:
       "eventType": "ALTER_TABLE",
       "dbName": "some_db",
       "tableName": "new_some_table",
-      "oldTableName": "some_table"
+      "tableLocation": "s3://table_location",
+      "tableParameters": {
```

Comment:
I'd prefer we don't emit anything in this case, looks a bit weird to have this empty thing?

Reply:
This is under discussion above.

Reply:
Done.